### ISSUE TITLE
Split Nodes plugin enhancement

### ISSUE THREAD

**Issue by antares1470** [constellation-app/constellation#743]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Running the latest version of Constellation
	* Attached the ***Support Package*** via `Help` > `Support Package`
    * Checked the FAQs: https://github.com/constellation-app/constellation/wiki/FAQ
    * Checked that your issue isn't already filed: https://github.com/constellation-app/constellation/issues
    * Checked that there is not already a module that provides the described functionality: https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

Currently when you run the split nodes plugin, any transactions hanging off the old node will be attached to one of the new nodes formed and the other node(s) will have a transaction to this node. There are situations though where this way of graphing the split isn't the desired behaviour.

In this new way to graph the split nodes, transactions hanging off the old node would go to all the new nodes created (not just one of them) with no transactions between the new nodes. In the plugin, there should be a parameter to decide between the old and new way of graphing the split.

e.g. A sample graph
![graph1](https://user-images.githubusercontent.com/62311243/88501753-acc68680-d00f-11ea-8b05-bcc5f212305b.png)

The current behaviour of Split Nodes
![graph2](https://user-images.githubusercontent.com/62311243/88501781-c2d44700-d00f-11ea-8f3c-7685c8773b30.png)

The suggested alternative behaviour
![graph3](https://user-images.githubusercontent.com/62311243/88501783-c5cf3780-d00f-11ea-9bc5-d2963492b946.png)

### Additional Information

If a user selects the new way to display the split, the Transaction Type parameter should be disabled as this won't be needed

**Comment by Auriga2** [comment]
I need to use `edit` method instead of the `query` because the `RecordStoreQueryPlugin` only sends details of the "selected" nodes (transactions or all) in the`RecordStore`. To implement the new functionality I need to know all the transactions adjacent to the selected node.

I guess if we choose to  split the below node highlighted in the left, it should result the graph at the right:

![image](https://user-images.githubusercontent.com/64826928/89359156-470b8600-d704-11ea-8c4c-a68e95dca92a.png)

**Comment by arcturus2** [comment]
@Auriga2 agreed, using the `edit` method is what I would use too. Thank you for the diagram, it makes it so much easier to understand; and that diagram to the right is what this issue is expecting to have.

A minor note is that I think you are just using the draw mode to create the graph on the right so transaction colours are randomly generated but just for completeness, the transaction colour from ghi -> vertex #3 should be yellow.

**Comment by Auriga2** [comment]
@arcturus2 Agree, I managed to draw them in  the same colour. 
I made these assumptions but let me know if any change is required:
- The  Id of the new transaction is set as the new transaction "Identifier". I could copy this attribute also, but felt like it should be unique. 
- graph.getVertexTransactionCount(mergedElement) returns the correct transactions count of a Node. Because when I was testing in debug mode, occasionally  some nodes were getting 0, although I could see lines connected to them. Then I saved the same graph and reopened it and the same nodes worked! 
- The existing functionality arranges all new new nodes in a grid. New option also follows that. But when multiple nodes are splitting in one go, this arranges them intermingled as in the below `Pic 1`. But it would be nice if we can make it similar to `Pic 2` (get this arrangement when splitting nodes one after the other manually). I tried `BUBBLE_TREE`, `TREES` etc, instead of `GRID_COMPOSITE` in `completionArrangement` to no avail. Arranging after each node didn't fix it either, because apparently the graph is actually arranged in the `SimpleQueryPlugin`, with all the changes. Maybe an expert can give this a try if needed.

Pic 1
![image](https://user-images.githubusercontent.com/64826928/89762003-b7643e00-db2e-11ea-8fcd-a4e8ae9ff56f.png)

Pic 2
![image](https://user-images.githubusercontent.com/64826928/89762176-0dd17c80-db2f-11ea-8491-af553993c5e0.png)
